src/request/export/ExportFile.js

```diff
@@ -66,11 +66,7 @@ class ExportFile extends Nimiq.Observable {
                 hideCancel: true,
             },
         );
-        this._passwordSetterBox = new PasswordSetterBox(
-            $passwordSetterBox, {
-                hideSkip: true,
-            },
-        );
+        this._passwordSetterBox = new PasswordSetterBox($passwordSetterBox);
```

Comment:
There should be no need for a password setter during export anymore, when keys cannot be unencrypted anyway. But we might want to keep it in anyway, to be sure?

Reply:
As discussed previously, I did not remove the functionality. It is just no longer reachable.